src/protocol-democracy.md

```diff
@@ -98,3 +98,60 @@ If we want to relax this in the future, we would need to come up with a way to h
 
 In order to determine if a vote is passing, we use Positive Turnout Bias. In addition we enforce a minimum turnout of 5%.
 
+### Documentation
+Go to the encointer-node repo and open 2 terminals:
+
+Terminal 1:
+```cargo build --release
+./target/release/encointer-node-notee --dev --enable-offchain-indexing true -lencointer=debug,parity_ws=warn
```

Comment:
As your node with the new democracy pallet is still on a branch, please supply the git and build  commands as well